freeipa/src/main/java/com/sequenceiq/freeipa/util/CrnService.java

```diff
@@ -69,4 +69,12 @@ public String createCrn(String accountId, Crn.ResourceType resourceType) {
                 .setResource(UUID.randomUUID().toString())
                 .build().toString();
     }
+
+    public static String checkUserCrn(String userCrn) {
+        if (userCrn == null) {
```

Comment:
This method doesn't really make much sense to me. It's not doing anything except a null-check.Also, why is it static?

I think it would make more sense to make this
`public Crn safeFromString(String userCrn)`
Then you can also call this from `getCurrentAccountId` and `getCurrentUserId` to ensure that they follow the same exception-handling behavior